test/unit/ui/app/actions.spec.js

```diff
@@ -331,7 +331,7 @@ describe('Actions', function () {
     })
 
     it('calls importAccountWithStrategies in background', async function () {
-      const store = mockStore()
+      const store = mockStore({ metamask: devState })
```

Comment:
This was added so that these [two selectors would ](https://github.com/MetaMask/metamask-extension/pull/8312/files#diff-eebb93d0666faf7aa39f85d9f7efa804R1204) not blow up. I should replace this with a more minimal state.

Reply:
Got it. Definitely won't block on that, whatever you do!

Reply:
I've replaced these with a more minimal state (just the selected address).